src/windows/BatteryProxy.js

```diff
@@ -19,7 +19,38 @@
  *
  */
 
-/* global WinJS, BatteryStatus */
+/* global Windows, WinJS, BatteryStatus */
+
+var PowerManager = Windows && Windows.System &&
+    Windows.System.Power && Windows.System.Power.PowerManager;
+
+if (PowerManager) {
+    var BatteryWin10 = {
+        start: function (win, fail) {
+
+            function reportStatus() {
+                win({
+                    level: PowerManager.remainingChargePercent,
+                    isPlugged: PowerManager.powerSupplyStatus !== Windows.System.Power.PowerSupplyStatus.notPresent
+                }, { keepCallback: true });
+            }
+
+            PowerManager.onremainingchargepercentchanged = reportStatus;
```

Comment:
Would not it be better to use add/remove event listener here instead to avoid overriding a listener which might be set in an app code?